### PR TITLE
Move content to Contributing guide and replace/fix images

### LINKED ISSUE #2005: Move these style recommendations to our contributing guide
[Link to internal project](https://www.notion.so/dbtlabs/Fast-follows-Docs-IA-refresh-cfe834d2530d43f2adfef242188b911d)

### Contributions

- [ ] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Move the content from these two page to our README contributing sections. Make sure to update redirects when we delete these pages!

- [ ] https://docs.getdbt.com/styles  we need to follow up with the Front end team to see what this page does!
- [x] https://docs.getdbt.com/useful_components

### What part(s) of the page would you like to see updated?

All

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by runleonarun**
fixes #2005 

## Description & motivation
This PR is a bit of a catchall for a few things I've come across lately. 

* Replaces some old images and fixing OAuth images that were incorrectly linked. 
* Moves content per #2005 
* Helps customers confused by the Project settings being buried in the UI. 

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md [hunk 1]
```diff
@@ -4,22 +4,28 @@ id: "artifacts"
 description: "Use artifacts to power your automated docs site and source freshness data." 
 ---
 
-## dbt Cloud Artifacts
+When dbt Cloud runs dbt jobs, it generates and saves *artifacts*. You can use these artifacts, like `manifest.json`, `catalog.json`, and `sources.json` to power different aspects of dbt Cloud, namely: [dbt Docs](documentation) and [source freshness reporting](cloud-snapshotting-source-freshness).
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md]:
```suggestion
When running dbt jobs, dbt Cloud generates and saves *artifacts*. You can use these artifacts, like `manifest.json`, `catalog.json`, and `sources.json` to power different aspects of dbt Cloud, namely: [dbt Docs](documentation) and [source freshness reporting](cloud-snapshotting-source-freshness).
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md [hunk 2]
```diff
@@ -4,22 +4,28 @@ id: "artifacts"
...
 
-When dbt Cloud runs dbt jobs, it generates and saves *artifacts*. These artifacts, like `manifest.json`, `catalog.json`, and `sources.json` are used to power different aspects of dbt Cloud, namely: [dbt Docs](documentation) and [source freshness reporting](cloud-snapshotting-source-freshness).
+## Create dbt Cloud Artifacts
 
-While every dbt Cloud job will produce artifacts, typically you should designate only one production job for a given project. When you edit a project and select the production job, dbt Cloud will render links to the production documentation and source freshness artifacts generated for that project. You can see these artifacts in the Run Overview page for the production job.
+While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the jobs that will produce documentation and source freshness artifacts.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md]:
```suggestion
While running any job can produce artifacts, you should only associate one production job with a given project to produce the project's artifacts. You can designate this connection in the **Project details** page. To access this page, click the gear icon in the upper right, select **Account Settings**, select your project, and click **Edit** in the lower right. Under **Artifacts**, select the jobs you want to produce documentation and source freshness artifacts for. 
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md [hunk 3]
```diff
@@ -4,22 +4,28 @@ id: "artifacts"
...
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/project-level-artifact-updated.png" title="Configuring Artifacts"/>
 
+If you don't see your job listed, you might need to edit the job and select **Run source freshness** and **Generate docs on run**.
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/edit-job-generate-artifacts.png" title="Editing the job to generate artifacts"/>
+
+When you add a production job to a project, dbt Cloud will update the content and provide links to the production documentation and source freshness artifacts generated for that project. You can see these links by clicking **Deploy** in the upper left, selecting **Jobs**, and then selecting the production job. From the job page, you can select a specific run to see how artifacts were updated for that run only.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md]:
```suggestion
When you add a production job to a project, dbt Cloud updates the content and provides links to the production documentation and source freshness artifacts it generated for that project. You can see these links by clicking **Deploy** in the upper left, selecting **Jobs**, and then selecting the production job. From the job page, you can select a specific run to see how artifacts were updated for that run only.
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md [hunk 4]
```diff
@@ -4,22 +4,28 @@ id: "artifacts"
...
+
 ### Documentation
 
-Once you select a job in your project settings, dbt Cloud will update "Documentation" in the top left so it links to documentation for this job. This link will always point to the latest version of the documentation for your account!
+Once set up, dbt Cloud will update the **Documentation** link in the upper left so it links to documentation for this job. This link will always point to the latest version of the documentation for your account!
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md]:
```suggestion
When set up, dbt Cloud updates the **Documentation** link in the upper left so it links to documentation for this job. This link always points to the latest version of the documentation for your account!
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md [hunk 5]
```diff
@@ -4,22 +4,28 @@ id: "artifacts"
...
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/doc-menu.png" title="A link to the latest documentation for the selected job"/>
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/doc-menu.png" title="A link to the latest documentation for the selected job"/> 
 
 ### Source Freshness
 
-As with Documentation, configuring a job for the Source Freshness artifact setting will update the "Data Sources" link under "Deploy" to point to the latest Source Freshness report for the selected job.
+As with Documentation, configuring a job for the Source Freshness artifact setting will update the Data Sources link under **Deploy** to point to the latest Source Freshness report for the selected job.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/artifacts.md]:
```suggestion
As with Documentation, configuring a job for the Source Freshness artifact setting also updates the Data Sources link under **Deploy** to point to the latest Source Freshness report for the selected job.
```

**Review by nghi-ly — COMMENTED** [review]
looks good! minor nits!

**Pull request closed without merging**